impl/sql/mutationstorage/queue.go

```diff
@@ -82,7 +83,7 @@ func (r *Receiver) Close() {
 
 // Flush sends any waiting queue items.
 func (r *Receiver) Flush() {
-	r.more <- true
+	r.sendBatch(context.Background(), true)
```

Comment:
This wants to be `context.TODO()` because you're not in the `main()` of any program here.

Reply:
That said, can `Flush()` take a context?

Reply:
Flush could probably take a context.